sites.yml

```diff
@@ -0,0 +1,31 @@
+- title: New Relic Developer Site
+
+  # this is the URL that is linked from the showcase
+  main_url: https://developer.newrelic.com/
+
+  # this URL is used to generate a screenshot
+  url: https://github.com/newrelic/developer-website/screenshots
```

Comment:
I was curious if this needs to reference the actual file in the path, the example they gave didn't require the file name.